claim/netdata-claim.sh.in

```diff
@@ -83,7 +83,7 @@ get_config_value() {
     config_result=$(@sbindir_POST@/netdatacli 2>/dev/null read-config "$section|$key_name"; exit $?)
     # shellcheck disable=SC2181
     if [ "$?" != "0" ]; then
-       echo 2> "cli failed, assume netdata is not running and query the on-disk config"
+       echo -n 2> "cli failed, assume netdata is not running and query the on-disk config"
```

Comment:
This creates a file with a name `cli failed, assume netdata is not running and query the on-disk config`. It should be `>&2` instead of `2>` like the rest of the `echo` commands. Also, I don't understand why we need `-n` here.

Reply:
Yes, the lack of >&2 is a typo. The need for the -n is a bit strange. Without it there is a stray newline that ends up at the beginning of stdout, so it is prepended to the start of the variable that captures the config value.

Reply:
I figured it out, the problem was the typo. Since instead of passing an argument to `echo` it passed a filename for redirection, then it was as if it only ran:
```
echo ""
```
However, only `stderr` was redirected with `2>` to the filename, so the actual `stdout` output of echo was exactly `\n`, which was prepended to the `stdout` contents of `get_config_value()` as a function.

By adding `-n` it fixes the problem, but fixing the typo negates the need for `-n`.

Reply:
That makes sense and I've just looked at the mishmash of redirects in the script that I will now make consistent, but why are are using `>&2` everywhere instead of `1>&2` ? It seems to work but I can't find that default listed in the bash manual.

Reply:
Of course, the default is stdout :). First coffee of the day is still working it's way through me...

Reply:
`>` is short for `1>` and it's implied that you're redirecting `stdout`. They are synonymous.